pkg/gatewayserver/scheduling/scheduler.go

```diff
@@ -447,3 +447,22 @@ func (s *Scheduler) TimeFromTimestampTime(t uint32) (ConcentratorTime, bool) {
 	}
 	return s.clock.FromTimestampTime(t), true
 }
+
+// SubBandStats returns a map with the usage stats of each sub band.
+func (s *Scheduler) SubBandStats() []*ttnpb.GatewayConnectionStats_SubBandStats {
+	var res []*ttnpb.GatewayConnectionStats_SubBandStats
+
+	for _, sb := range s.subBands {
+		if sb == nil {
+			continue
+		}
```

Comment:
Why?

Reply:
To avoid nil subbands breaking this, but it should never happen anyway. Removed.